manager/state/raft/membership/cluster.go

```diff
@@ -180,7 +181,7 @@ func (c *Cluster) ReplaceMemberConnection(id uint64, oldConn *Member, newConn *M
 		return ErrIDNotFound
 	}
 
-	if oldConn.Conn != oldMember.Conn {
+	if !force && oldConn.Conn != oldMember.Conn {
```

Comment:
The normal use case for this function is to bounce the connection. I'm extending it here to also allow a change of address. Maybe it would be less confusing to have a different function for address changes.